resources/fetchers/ecr_factory_test.go

```diff
@@ -68,7 +68,7 @@ default_region: us1-east
 			"us1-east",
 			"my-account",
 			[]string{
-				"^my-account\\.dkr\\.ecr\\.us1-east\\.amazonaws\\.com\\/([-\\w\\.\\/]+)[:,@]?",
+				"^my-account\\.dkr\\.ecr\\.([-\\w]+)\\.amazonaws\\.com\\/([-\\w\\.\\/]+)[:,@]?",
 				"public\\.ecr\\.aws\\/\\w+\\/([-\\w\\.\\/]+)\\:?",
```

Comment:
can you please add some documentation about this regex, what are we expecting it to parse, and an example string or something?